src/System.Memory/src/System/Buffers/BuffersExtensions.cs

```diff
@@ -78,6 +78,9 @@ public static void CopyTo<T>(in this ReadOnlySequence<T> source, Span<T> destina
 
         private static void CopyToMultiSegment<T>(in ReadOnlySequence<T> sequence, Span<T> destination)
         {
+            if (sequence.Length > destination.Length)
```

Comment:
Should this check be moved into the `else` of the public method for callstack reasons?

Reply:
The public method is aggressive inlining. I'd rather this show up in the call stack than pay the cost at every call site.